integration/integration_cnb_test.go

```diff
@@ -57,16 +58,24 @@ func TestCNBIntegrationNPMProject(t *testing.T) {
 	})
 	defer container2.terminate(t)
 
-	err := container.whenRunningPiperCommand("cnbBuild", "--noTelemetry", "--verbose", "--path", "TestCnbIntegration/project", "--customConfig", "TestCnbIntegration/config.yml", "--containerImageName", "node", "--containerImageTag", "0.0.1", "--containerRegistryUrl", registryURL)
+	err := container.whenRunningPiperCommand("cnbBuild", "--noTelemetry", "--verbose", "--path", "TestCnbIntegration/project", "--customConfig", "TestCnbIntegration/config.yml", "--containerImageName", "node", "--containerImageTag", "0.0.1", "--containerRegistryUrl", registryURL, "--defaultProcess", "greeter")
 	assert.NoError(t, err)
+	buffer, err := container.getPiperOutput()
+	assert.NoError(t, err)
+	fmt.Println("logs:")
+	buffer.WriteTo(os.Stdout)
+	fmt.Println("\nend of logs")
 	container.assertHasOutput(t, "running command: /cnb/lifecycle/creator")
 	container.assertHasOutput(t, "Selected Node Engine version (using BP_NODE_VERSION): 16")
 	container.assertHasOutput(t, "Paketo NPM Start Buildpack")
 	container.assertHasOutput(t, fmt.Sprintf("Saving %s/node:0.0.1", registryURL))
+	container.assertHasOutput(t, "Setting default process type 'greeter'")
 	container.assertHasOutput(t, "*** Images (sha256:")
 	container.assertHasOutput(t, "SUCCESS")
 	container.terminate(t)
 
+	fmt.Print("\n--- Test 1 done ---\n\n")
```

Comment:
Is this a leftover?

Reply:
For debugging the problem in the integration test.

I guess we should simply drop this commit from the branch: https://github.com/SAP/jenkins-library/pull/4140/commits/81aceb81b0617db5c1b7db69328c0357dd4dd42c